src/pages/developers.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+import Developers from '../components/developers/developers.component'
+
+const DevelopersPage = () => {
+  return (
+    <Developers />
```

Comment:
Instead of having a page that just returns the component, why not add the component here directly? Seems like an unnecessary abstraction

Reply:
@elias-garcia Correct me if I'm wrong, routing in Gatsbyjs requires the use of pages but on the other hand we wanted to abstract components and styling from there. If I remember correctly this is what Elias and me talked at the start.

Reply:
Okay, not a fan of these kind of abstractions, but let's leave it

Reply:
Well, not exactly. The problem is that inside the Gatsby `pages` folder you cannot have nested subfolders, so `styles` files would be siblings of the `jsx` page files. But I don't think that styles should be in that folder, so what I tipically do is to abstract pages with components which already have styles, for example:

```js
function Page() {
  return (
    <HeroBanner />
    <Section1 />
    <Section2 />
    <Contact />
  </>
}
```

Instead of returning just a one big component. But I didn't said nothing because we can refactor this at any time, it shouldn't take too much time.